.github/workflows/build.yml

```diff
@@ -31,3 +40,42 @@ jobs:
 
       - name: Build
         run: cargo build --release --locked
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4
+        with:
+           name: nrr-${{ matrix.system.arch }}-${{ matrix.system.name }}
+           path: ./target/release/nrr*
+
+  linux-static:
+    strategy:
+      matrix:
+        arch: 
+          - x86_64
+          - arch64
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Nix
+        uses: DeterminateSystems/nix-installer-action@v10
+
+      - name: Setup Nix cache
+        uses: determinatesystems/magic-nix-cache-action@v4
+
+      - name: Build package
+        run: |
+          nix build .#nrr-static-${{ matrix.arch }} \
+            --accept-flake-config \
```

Comment:
I would prefer if it didn't use things from the garnix cache; after all, the Magic Nix Cache is already present.
```suggestion
```